awx_collection/tests/integration/targets/tower_project_update/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Update a project without waiting
+  tower_project_update:
+    name: "AWX-Collection-tests-tower_job_template-proj-{{ test_id }}"
```

Comment:
maybe I missed something, but doesn't a project with this name need to be created first?

Reply:
It does need to be created, which is why I took the info from the tower project job thinking it would be run after.